doc: add leading asterisks for javadoc

Gaurav-Punjabi opened the pull request:
Fixes the violations that are generated after solving issue #6722 
PR Link for Check: [MissingLeadingAsterisk](https://github.com/checkstyle/checkstyle/pull/8085)

strkkk commented on src/main/java/com/puppycrawl/tools/checkstyle/gui/TreeTable.java:
```diff
@@ -184,7 +184,8 @@ public void updateUI() {
 
-    /* Workaround for BasicTableUI anomaly. Make sure the UI never tries to
+    /**
+     * Workaround for BasicTableUI anomaly. Make sure the UI never tries to
```
I dont think this is a valid fix. MissingAsterisk checks only javadocs, but it is not a javadoc comment.
Same for other cases below

Gaurav-Punjabi replied:
Done, updated.

Gaurav-Punjabi commented:
> Please rename commit message, now it is incorrect.

Done :)

strkkk commented:
ok to merge after ci pass

Gaurav-Punjabi commented:
@strkkk New violations were generated, after updating the check according to the following comment [checkstyle/checkstyle#8085](https://github.com/checkstyle/checkstyle/pull/8085#discussion_r410207230). So I fixed them and updated the PR. Please review.
